+++
title = "Complex Genomics Analysis Pipelines made Simple with NextFlow & AWS Batch"
date = 2019-02-27T15:00:00  # Schedule page publish date.
draft = false

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
time_start = 2019-02-27T15:00:00
#time_end = 2030-06-01T15:00:00

# Abstract and optional shortened version.
abstract = "The science of genomics is increasingly contributing to the sort of insights which lead to longer and healthier lives for us all. But the computational pipelines needed to support accurate and timely analysis of huge quantities of genomic data are truly extreme - more than enough to break the most well configured on-premise HPC facility. The key factor? A lack of elasticity: no ability to stretch the infrastructure a million different ways to accommodate needs that change from minute to minute and just aren't predictable. When this happens, whole collections of analysis jobs get piled up behind each other, similar to a traffic jam on a complex road network. We can't fix the traffic on the roads, but for genomics, we have tools like NextFlow.io (a popular open source data-driven pipeline tool for orchestrating scientific pipelines) and AWS Batch, which have access to Amazon EC2 and all the elasticity a scientist could want. Thanks to all these technologies, our customer QBiC Tübingen is able to put all their efforts into their number one mission: to improve patient health by translating our understanding of the Human Gut Microbiome into knowledge and techniques for creating innovative new medicines that will help us all. You'll learn about: • QBiC's use of NextFlow's to manage complex pipelines and orchestrate interdependent jobs; • Their reasons for choosing AWS and the steps they took to build trust in us; • NextFlow's integration with AWS Batch which dynamically expands and contracts cloud resources to meet the needs of the pipeline as it works its way through the data. Suited for: HPC Users, Medical IT and the engineers who support them."
abstract_short = ""

# Name of event and optional event URL.
event = "Invited Talk at AWS Summit 2019 Berlin"
event_url = ""

# Location of event.
location = "Station Berlin, Luckenwalder Str. 4-6, 10963 Berlin"

# Is this a selected talk? (true/false)
selected = false

# Projects (optional).
#   Associate this talk with one or more of your projects.
#   Simply enter your project's filename without extension.
#   E.g. `projects = ["deep-learning"]` references `content/project/deep-learning.md`.
#   Otherwise, set `projects = []`.
projects = []

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["reproducible research", "Bioinformatics", "Nextflow", "Nf-core", "AWS", "Cloud", "Cloud Computing"]

# Links (optional).
url_pdf = ""
url_slides = "https://aws-de-marketing.s3-eu-central-1.amazonaws.com/Field%20Marketing/Summit-Berlin-2019/Presentations/AWS_Summit_Berlin_2019_Feb27_Complex%20Genomics%20Analysis%20Pipelines%20made%20Simple%20with%20NextFlow%20&%20AWS%20Batch.pdf"
url_video = ""
url_code = "https://nf-co.re"

# Does the content use math formatting?
math = false

# Does the content use source code highlighting?
highlight = false

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
#[header]
#image = "headers/bubbles-wide.jpg"
#caption = "My caption :smile:"

+++
